lib/syncthing/syncthing.go

```diff
@@ -77,6 +77,7 @@ type App struct {
 	stopOnce          sync.Once
 	mainServiceCancel context.CancelFunc
 	stopped           chan struct{}
+	M                 model.Model
```

Comment:
Let's be a little bit more verbose in the naming here...

```suggestion
	Model             model.Model
```